client/src/components/Modal/MapModal.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Modal, TouchableOpacity, Dimensions, TouchableWithoutFeedback, Text } from 'react-native';
+
+import MapView, { Marker } from 'react-native-maps';
+import { IconMarker } from '../../components/MapMarker/';
+
+import styles from './styles';
+
+const screen = Dimensions.get('window');
+
+const ASPECT_RATIO = screen.width / screen.height;
+const LATITUDE_DELTA = 0.0922;
+const LONGITUDE_DELTA = LATITUDE_DELTA * ASPECT_RATIO;
+
+const mapModal = ({
```

Comment:
`MapModal` please - this name does actually show up in the component hierarchy I'm pretty sure so it's important to capitalise it.

Reply:
Nice catch, thanks

Reply:
Nice catch, thanks